Dfe.Academies.Academisation.Domain/Validations/DeleteApplicationValidator.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Dfe.Academies.Academisation.Domain.ApplicationAggregate;
+using FluentValidation;
+
+namespace Dfe.Academies.Academisation.Domain.Validations;
+
+internal class DeleteApplicationValidator : AbstractValidator<int>
+{
+	public DeleteApplicationValidator()
+	{
+		RuleFor(x => x).
```

Comment:
We should validate that an Application is not already deleted, this is an edge case but possible if Id's are erroneous.

Reply:
we can ..but it cant be done in the validator layer as we'd need to make a call from here

Reply:
its not a full 'update' as such.. the end point posts a single application id to the db...the validation is triggered at the  domain layer application.cs  > public CommandResult Delete(int applicationId)   it would need to be changed to  public CommandResult Delete(application application)